src/arrayMethodJoin.js

```diff
@@ -4,8 +4,33 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    const joinedArr = [];
```

Comment:
you'd rather give this array the following name
```suggestion
    const splitArray = [];
```
it consists of items of array